src/components/tabs/TabInterface.ts

```diff
@@ -0,0 +1,7 @@
+import { Vue } from 'vue-property-decorator';
+
+interface TabInterface extends Vue {
```

Comment:
Why do we need this `TabInterface`? I thought we fixed this last time?

Reply:
Oh yeah, I remember we did get rid of it. I forgot because WebStorm throws errors for me...
![image](https://user-images.githubusercontent.com/47478125/96374275-a3aa0900-1169-11eb-870c-7a14ae7e72c2.png)

Reply:
Since the error only seems to appear in WebStorm, and doesn't come up when I do npm run serve, I've deleted it again.

Reply:
This is really weird. Worst case can you try doing `(tab as Tab).setVisible(index === given);`?

Reply:
I still get the same error, that "setVisible" is not a property of Vue.